.github/workflows/pr.yaml

```diff
@@ -53,3 +54,12 @@ jobs:
       # This selects "ARCH=amd64 + the latest supported Python + CUDA".
       matrix_filter: map(select(.ARCH == "amd64")) | max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]) | [.]
       script: "ci/build_wheel.sh"
+  wheel-tests:
+    needs: wheel-build
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-25.02
+    with:
+      build_type: pull-request
+      script: "ci/test_wheel.sh"
+      # This selects "ARCH=amd64 + the latest supported Python + CUDA".
+      matrix_filter: map(select(.ARCH == "amd64")) | max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]) | [.]
```

Comment:
```suggestion
```

I think we should cover the full RAPIDS PR CI matrix here.

Even though the `dask-cuda` wheels are pure-Python and not CPU architecture or GPU-version dependent, with tags like this:

```text
dask_cuda-25.2.0a10-py3-none-any.whl
```

... some of its dependencies do produce different variants for those different environments, and it's possible `dask-cuda` could interact with them in different ways that we'd want to catch. I'm reminded for example of https://github.com/microsoft/LightGBM/issues/6509 and https://github.com/microsoft/LightGBM/pull/6654, where I found that there was some aarch64-specific issue whose resolution was "load OpenMP as early as possible".

By the way, for PRs that matrix is only 2 jobs. You can find them here:

https://github.com/rapidsai/shared-workflows/blob/3313dce51b2cfea4601c30a986cd5980087fe2c1/.github/workflows/wheels-test.yaml#L81-L85